front/src/js/service.js

```diff
@@ -0,0 +1,195 @@
+/*
+- add axios cdn link in index.html :
+     	<script src="https://unpkg.com/axios@0.21.1/dist/axios.min.js"></script>
+- call service.js in index.html :
+    <script src="/front/js/service.js"></script>
+- start scratchy service : 
+    var api = new ScratchyService("http://localhost:5000/api");
+    api;
+- call a function :
+    api.function();|
+*/
+
+function ScratchyService(apiUrl) {
+    let _this = this;
+    this.cacheUsers = []; // List of users, used as cache
+
+//                               ROOM FUNCTIONS
+
+
+    /**
+     * @typedef {Object} Room
+     * @param {string} roomID - room id , eg:60895dd62d1a706830c31f10
+     * @param {string} roomTitle - title of a room , eg:my room
+     * @param {string} roomDescription - description of a room, eg:my room description
+     * @returns {string} - room id, title and description
+     */
+    this.updateRoom = async function(roomId,roomTitle,roomDescription){
+        const reponse = await axios.put(apiUrl+'/room/'+roomId, { title: roomTitle , description: roomDescription });
+        return reponse.data;
+    };
+
+    /**
+     * 
+     * @typedef {Object} Room
+     * @param {string} roomTitle - title of a room, eg:my room
+     * @param {string} roomDescription - description of a room, eg:my room description
+     * @returns {string} - room id 
+     */
+    this.createRoom = async function(roomTitle,roomDescription){
+        const reponse = await axios.post(apiUrl+'/room', { title: roomTitle , description: roomDescription });
+        return reponse.data;
+    };
+
+    /**
+     * 
+     * @typedef {Object} Room
+     * @param {string} roomID - room id , eg:60895dd62d1a706830c31f10
+     * @returns {Room} room information , { oid:"60895dd62d1a706830c31f10" ,title:"example"}
+     */
+    this.getRoom = async function(roomID){ // put the id between "" 
+            // Make a request for a user with a given ID
+            let reponse = await axios.get(apiUrl+'/room/'+roomID);// make the GET request
+            return reponse.data; // return data JSON           
+    };
+
+    
+    /**
+     * 
+     * @typedef {Object} Room
+     * @param {string} roomID - room id, eg:60895dd62d1a706830c31f10
+     */
+     this.deleteRoom = async function(roomID){
+        const reponse = await axios.delete('http://localhost:5000/api/room/'+roomID);
```

Comment:
delete function doesn't send back anything

Reply:
why you want i delete this function ? `const reponse = await axios.delete('http://localhost:5000/api/room/'+roomID);` make just a delete request that not the response.

Reply:
no i mean you can make
```js
await axios.delete('http://localhost:5000/api/room/'+roomID);
return None
```
since the delete function doesn't send back anything